src/main/java/com/blamejared/brackets/util/IMaterialDefinition.java

```diff
@@ -0,0 +1,20 @@
+package com.blamejared.brackets.util;
+
+import stanhebben.zenscript.annotations.*;
+
+/**
+ * Created by Jared on 6/16/2016.
+ */
+@ZenClass("modtweaker.material.IMaterialDefinition")
+public interface IMaterialDefinition {
+
+    @ZenOperator(OperatorType.MUL)
```

Comment:
I'm suprised this even works...

```
OperatorType.MUL
```
is used for stacks of amounts of items / liquids / etc

so like:
```
<minecraft:dirt> * 5
``` 
for a stack of 5 dirt
or
```
<liquid:water> * 80
```
for a liquid stack of 80Mb

So having it here is useless imo